example/index.html

```diff
@@ -8,6 +8,15 @@
   <main id="app"></main>
   <main id="app-increase-2"></main>
 
-  <script type="text/javascript" src="builds/main.js"></script>
+  <script type="text/javascript" id="app-loader"></script>
```

Comment:
Use HTML Webpack Plugin to auto-generate the index.html file. No custom JS should be needed.